### PR TITLE
DOC-6018 add note about incremental backups being dropped

### PR DESCRIPTION
**Pull request by ashleykwtam**
Addresses DOC-6018:
- Associated scheduled incremental backups are dropped when `DROP SCHEDULES` is called.
- Added this to the [`DROP SCHEDULES`](https://deploy-preview-17777--cockroachdb-docs.netlify.app/docs/stable/drop-schedules) and [Manage a Backup Schedule](https://deploy-preview-17777--cockroachdb-docs.netlify.app/docs/v23.2/manage-a-backup-schedule#drop-the-schedule) page

### CODE REVIEW

File: src/current/v23.1/drop-schedules.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ docs_area: reference.sql
 `DROP SCHEDULE` does **not** cancel any in-progress jobs started by the schedule. Before you drop a schedule, [cancel any in-progress jobs]({% link {{ page.version.version }}/cancel-job.md %}) first, as you will not be able to look up the job ID once the schedule is dropped.
 {{site.data.alerts.end}}
 
+When `DROP SCHEDULES` removes a [scheduled full backup]({% link {{ page.version.version }}/create-schedule-for-backup.md %}#create-a-schedule-for-full-backups-only-core), all associated [scheduled incremental backups]({% link {{ page.version.version }}/create-schedule-for-backup.md %}#incremental-backup-schedules) are dropped.
```
Comment (msbutler) [inline src/current/v23.1/drop-schedules.md:14]: the backup itself is not dropped, rather the schedule is dropped

I'd rewrite this as "When `DROP SCHEDULES` removes a full backup schedule, it removes the associated incremental backup schedule, if it exists"

**Review by MichaelTrestman — COMMENTED** [review]
LGTM

File: src/current/v23.1/drop-schedules.md [hunk 2]
```diff
@@ -11,6 +11,8 @@ docs_area: reference.sql
 `DROP SCHEDULE` does **not** cancel any in-progress jobs started by the schedule. Before you drop a schedule, [cancel any in-progress jobs]({% link {{ page.version.version }}/cancel-job.md %}) first, as you will not be able to look up the job ID once the schedule is dropped.
 {{site.data.alerts.end}}
 
+When `DROP SCHEDULES` removes a [full backup schedule]({% link {{ page.version.version }}/create-schedule-for-backup.md %}#create-a-schedule-for-full-backups-only-core), it removes the associated [incremental backup schedules]({% link {{ page.version.version }}/create-schedule-for-backup.md %}#incremental-backup-schedules), if it exists.
```
Comment (msbutler) [inline src/current/v23.1/drop-schedules.md:14]: nit: s/schedules/schedule/r

File: src/current/v23.1/manage-a-backup-schedule.md [hunk 3]
```diff
@@ -135,7 +135,7 @@ Or nest a [`SELECT` clause]({% link {{ page.version.version }}/select-clause.md
 > DROP SCHEDULES WITH x AS (SHOW SCHEDULES) SELECT id FROM x WHERE label = 'schedule_database';
 ~~~~
 
-For more information, see [`DROP SCHEDULES`]({% link {{ page.version.version }}/drop-schedules.md %}).
+When `DROP SCHEDULES` removes a [scheduled full backup]({% link {{ page.version.version }}/create-schedule-for-backup.md %}#create-a-schedule-for-full-backups-only-core), all associated [scheduled incremental backups]({% link {{ page.version.version }}/create-schedule-for-backup.md %}#incremental-backup-schedules) are dropped. For more information, see [`DROP SCHEDULES`]({% link {{ page.version.version }}/drop-schedules.md %}).
```
Comment (msbutler) [inline src/current/v23.1/manage-a-backup-schedule.md:138]: nit: this block still needs to be changed.

File: src/current/v23.2/manage-a-backup-schedule.md [hunk 4]
```diff
@@ -135,7 +135,7 @@ Or nest a [`SELECT` clause]({% link {{ page.version.version }}/select-clause.md
 > DROP SCHEDULES WITH x AS (SHOW SCHEDULES) SELECT id FROM x WHERE label = 'schedule_database';
 ~~~~
 
-For more information, see [`DROP SCHEDULES`]({% link {{ page.version.version }}/drop-schedules.md %}).
+When `DROP SCHEDULES` removes a [scheduled full backup]({% link {{ page.version.version }}/create-schedule-for-backup.md %}#create-a-schedule-for-full-backups-only-core), all associated [scheduled incremental backups]({% link {{ page.version.version }}/create-schedule-for-backup.md %}#incremental-backup-schedules) are dropped. For more information, see [`DROP SCHEDULES`]({% link {{ page.version.version }}/drop-schedules.md %}).
```
Comment (msbutler) [inline src/current/v23.2/manage-a-backup-schedule.md:138]: nit: this block still needs to be changed.

Comment (ashleykwtam) [reply src/current/v23.2/manage-a-backup-schedule.md]: fixed

**Review by msbutler — APPROVED** [review]
LGTM!

**Pull request merged**